### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

**Review by seanlip — APPROVED** [review]

File: app/src/main/res/values-pcm/strings.xml [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+  <plurals name="completed_story_count">
+    <item quantity="one">1 Story Completed\n    </item>
+    <item quantity="zero">%s Stories Completed\n    </item>
+    <item quantity="other">\n      %s Stories Completed\n    </item>
+  </plurals>
+  <plurals name="ongoing_topic_count">
+    <item quantity="one">1 Topic in Progress\n    </item>
+    <item quantity="zero">%s Topics in Progress\n    </item>
+    <item quantity="other">\n      %s Topics in Progress\n    </item>
+  </plurals>
```
Comment (adhiamboperes) [inline app/src/main/res/values-pcm/strings.xml:204]: There are incorrect new lines inserted in these plurals. Tested it out and they definitely display incorectly within the app.

File: app/src/main/res/values-pcm/strings.xml [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+  <plurals name="days">
+    <item quantity="one">a day</item>
+    <item quantity="other">%s days</item>
+  </plurals>
+  <string name="topic_revision_recyclerview_tag">topic_revision_recyclerview_tag</string>
+  <string name="ongoing_recycler_view_tag">ongoing_recycler_view_tag</string>
+  <string name="item_selection_text">Abeg select all di correct choices.</string>
+  <string name="unsupported_app_version_dialog_title">\n    Unsupported app version</string>
+  <string name="unsupported_app_version_dialog_message">\n    Dis version of di app no longer dey supported. Abeg update am from di Play Store.</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values-pcm/strings.xml:438]: Same new lines issue here

File: app/src/main/res/values-pcm/strings.xml [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+  <string name="beta_notice_dialog_title">Beta Notice</string>
+  <string name="beta_notice_dialog_message">Hello! Your app don dey update to di Beta version. If you experience any problems while you dey use di app, or get any questions, abeg contact us at android-feedback@oppia.org.</string>
+  <string name="beta_notice_dialog_do_not_show_again_text">No show dis message again</string>
+  <string name="beta_notice_dialog_close_button_text">OK</string>
+  <string name="general_availability_notice_dialog_title">General Availability Notice</string>
+  <string name="general_availability_notice_dialog_message">Hello! Your app don dey update to di General Availability version. If you experience any problems while you dey use di app, or get any questions, abeg contact us at android-feedback@oppia.org.</string>
+  <string name="general_availability_notice_dialog_do_not_show_again_text">No show dis message again</string>
+  <string name="general_availability_notice_dialog_close_button_text">OK</string>
+  <string name="ratio_content_description_separator">\n     to</string>
+  <string name="ratio_default_hint_text">\n    Enter a ratio in di form x:y.</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values-pcm/strings.xml:452]: New lines

File: app/src/main/res/values-pcm/strings.xml [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+  <string name="fraction_error_divide_by_zero">Abeg no put 0 for di denominator</string>
+  <string name="fraction_error_larger_than_seven_digits">None of di nombas for di fraction go get more dan 7 digits.</string>
+  <string name="number_error_starting_with_floating_point">Abeg start your ansa with nomba (e.g.,”0” for 0.5).</string>
+  <string name="number_error_invalid_format">Abeg put correct nomba.</string>
+  <string name="number_error_larger_than_fifteen_characters">Di ansa fit get at most 15 digits (0–9) or sign (. or -).</string>
+  <string name="ratio_error_invalid_chars">\n    Abeg write a ratio wey get nomba separated by colons (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_format">\n    Abge enter a valid ratio (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_colons">\n    Your ansa get two colons (:) next to each other.</string>
+  <string name="ratio_error_invalid_size">\n    Nomba of terms no dey equal to di required terms.</string>
+  <string name="ratio_error_includes_zero">\n    Ratios no suppose get 0 for inside.</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values-pcm/strings.xml:171]: In the main strings file, these strings span more than one line, hence the [incorrect] insertion of the newline.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Strings in this PR are broken. I can pinpoint the issue to strings that have a "description" attribute to them, making them span more than one line.

The same issue is also in the "plurals", again because they are multiline. I think translatewiki does not know how to intepret this.

See sample screenshots: 
| Correct | Incorrect |   
|---|---|
|Screenshot_1685619646|Screenshot_1685619632|

**Comment by BenHenning** [comment]
Note that this PR is being replaced by #5010 since we can't wait for the next auto-push of translations from Translatewiki (plus, that PR is introducing the actual support for Naija which leads to the CI checks passing).

**Pull request closed without merging**